src/midi/shared/timer.rs

```diff
@@ -1,6 +1,9 @@
 #![allow(dead_code)]
 
-use std::time::{Duration, Instant};
+use std::time::Instant;
+use time::Duration;
+
+pub const ROLLIN: Duration = Duration::seconds(2);
```

Comment:
Nitpick: in my other midi players, rollin is the distance from the top to the bottom of the view, so it changes based on view speed

Reply:
Definitely much harder to implement tho

Reply:
I could rename it to `START_DELAY` or something

Reply:
Yeah it's ok, not important anyway